plugins.vim

```diff
@@ -68,14 +52,12 @@ Plug 'gorkunov/smartpairs.vim'                       " Smart selection between p
 Plug 'itspriddle/vim-stripper'                       " Strip trailing whitespace on save
 Plug 'justincampbell/vim-eighties'                   " Ensures windows are at least 80 chars wide (my PR added the ability to ignore a list of additional bufnames)
 Plug 'scrooloose/nerdcommenter'                      " Comment multiple lines
-Plug 'tpope/vim-endwise'                             " Auto adds end to method definitions, blocks, etc
```

Comment:
Caused instability with CoC

Reply:
Would this help possibly? https://github.com/danieljaouen/vim-endwise/commit/fa93e76015a69939201a239bd2d00cdd6f7e5ea4